### DISCUSSION TITLE
What do you think about my Wittstock theme?

### DISCUSSION THREAD

**Discussion by schulle4u** [datenstrom/community#756] [Ask a question] [answered]
In an attempt to use more semantic HTML5 elements in Yellow's layouts, here is [Wittstock](https://github.com/schulle4u/yellow-wittstock), a theme extension using (mostly) classless layouts. This extension uses [Simple.css 2.0.0](https://github.com/kevquirk/simple.css) by Kev Quirk as underlying style sheet. 

Included are the default layouts, extension layouts from the official repository, and from my own extensions repository. Most things should work, however visual glitches are possible, especially when using dark mode. As I'm visually impaired, I'm only able to test things from a technical perspective. As always, feedback is very welcome.

If you are curious about the extension's name: I've decided to follow Datenstroms theme tradition. [This is Wittstock](https://en.wikipedia.org/wiki/Wittstock), the small town I'm living in. 🙂

**Comment by glow12121** [comment, marked as answer]
Well done, Steffen, thank you!

**Comment by schulle4u** [comment]
Here are some changes since the first release: 

* Theme moved into individual repository, see updated link in first post. 
* Updated to SimpleCSS v2.1.1
* Removed custom CSS file for simplicity and due to importing problems.
* Added custom Yellow classes (task list item, table of contents, notices, for light and dark mode to main CSS file (thanks Mark for help). 
* Slightly tweaked some templates, mostly accessibility-related.
